docs/roles/salt.md

```diff
@@ -6,7 +6,8 @@ Salt is used as a configuration management system, converging the state of all r
 
 ### Assigning roles to minions
 
-[Ubiquitous roles](../roles/) are collections of [Salt states](https://docs.saltstack.com/en/latest/topics/tutorials/starting_states.html) designed to be assigned to servers using [Salt grains](https://docs.saltstack.com/en/latest/topics/grains/). To add a role to a server with the Salt minion already installed, simply edit `/etc/salt/grains` with content along the lines of the following:
+[Ubiquitous roles](../roles) are collections of [Salt states](https://docs.saltstack.com/en/latest/topics/tutorials/starting_states.html) designed to be assigned to servers using [Salt grains](https://docs.saltstack.com/en/latest/topics/grains/).
+To add a role to a server with the Salt minion already installed, simply edit `/etc/salt/grains` with content along the lines of the following:
```

Comment:
I feel this belongs as a paragraph -- see the [GitBook rendered docs](https://lukecarrier.gitbooks.io/ubiquitous-moodle/roles/salt.html).